src/test/ui/hygiene/hygienic-labels.rs

```diff
@@ -10,38 +10,28 @@
 macro_rules! loop_x {
     ($e: expr) => {
         // $e shouldn't be able to interact with this 'x
-        'x: loop { $e }
-        //~^ WARNING shadows a label name that is already in scope
-        //~| WARNING shadows a label name that is already in scope
-        //~| WARNING shadows a label name that is already in scope
-        //~| WARNING shadows a label name that is already in scope
```

Comment:
Wait, this isn't a regression? The new behavior is what this should have always been, right?

Reply:
This is explicitly proposed to lang team in https://github.com/rust-lang/rust/pull/96296.